### PR TITLE
Admin to add and delete account

### PR DESCRIPTION
**Pull request by ODORA0**
Administrator adds and removes patient info

issue worked on https://issues.openmrs.org/browse/LUI-135

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ */
+/**
+ * 
+ */
+package org.openmrs.module.legacyui;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+//import org.openmrs.Patient;
+// import org.openmrs.propertyeditor.PatientEditor;
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java]: Had see this https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode?

File: api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+/**
+ * @author Admin
+ *
+ */
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java]: The above comment makes no sense

File: api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+public class LegacyUIModule {
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java]: Why did you introduce this new class, I find it redundant!

File: api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	}
+	
+	public void confirmDelete () {
+		log.info ("Are you sure you want to delete this location? It will be permanently removed from the system");
+	}
+	
+	public void purgedSuccessfully () {
+		log.info ("Location deleted Successfully");
+	}
+
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java]: Did you intend to make all the above methods logging methods. It's sort of trivial!

Comment (ODORA0) [reply api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java]: yup thats right, thought having them logging methods would be ok

File: api/src/main/java/org/openmrs/module/legacyui/LegacyUIActivator.java [hunk 5]
```diff
@@ -21,6 +21,8 @@
 public class LegacyUIActivator extends BaseModuleActivator {
 	
 	protected Log log = LogFactory.getLog(getClass());
+	
+
```
[the remark this answers is not in the document]
Comment (ODORA0) [reply api/src/main/java/org/openmrs/module/legacyui/LegacyUIActivator.java]: thats wasn't really intended though

File: api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java (hunk 1 above)
Comment (ODORA0) [reply api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java]: I'll definitely have a look at that

**Pull request closed without merging**